packages/aws-cdk-lib/custom-resources/test/aws-custom-resource/aws-custom-resource.test.ts

```diff
@@ -26,11 +26,13 @@ test('aws sdk js custom resource with onCreate and onDelete', () => {
       physicalResourceId: PhysicalResourceId.of('loggroup'),
     },
     onDelete: {
-      service: 'CloudWatchLogs',
-      action: 'deleteRetentionPolicy',
+      service: 'CloudWatch',
+      action: 'tagResource',
       parameters: {
-        logGroupName: '/aws/lambda/loggroup',
+        ResourceARN: 'dummy',
+        Tags: [{ Key: 'Name', Value: 'prod' }],
       },
+      physicalResourceId: PhysicalResourceId.of('add_tag'),
```

Comment:
Should we add a new test instead of modifying this one so that we can continue testing cloudwatch-logs?